cdap-proto/src/main/java/io/cdap/cdap/proto/security/NamespacePermission.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.proto.security;
+
+/**
+ * Permissions specifically  for CDAP Namespace.
+ */
+public enum NamespacePermission implements Permission {
+  /**
+   * Permission to read data in the repository of a namespace.
```

Comment:
nit: Please add that these permissions are specifically used for SCM, e.g. `read data in the SCM repository`, same for other docs, thanks!

Reply:
Added SCM in comments, thanks!